ansible_ai_connect/users/views.py

```diff
@@ -173,6 +174,7 @@ def get_context_data(self, **kwargs):
         context["project_name"] = settings.ANSIBLE_AI_PROJECT_NAME
         context["deployment_mode"] = settings.DEPLOYMENT_MODE
         context["has_active_plan"] = has_active_plan
+        context["days_left"] = int((has_active_plan.expired_at - timezone.now()).days)
```

Comment:
This will throw an exception if the User has no active plans, as `has_active_plan` will be `None`.

Reply:
If more time has elapsed than the plan duration this will show a negative figure. 

Would you rather limit to a minimum of `0`?

Reply:
Won't the call `has_active_plan.expired_at` fail due to the `None`.. way before you try to cast to an `int`?